pom.xml

```diff
@@ -53,7 +55,16 @@
             <artifactId>guava</artifactId>
             <version>${guava.version}</version>
         </dependency>
-
+        <dependency>
+            <groupId>com.github.f4b6a3</groupId>
+            <artifactId>uuid-creator</artifactId>
```

Comment:
do we really need a third party uuid lib?

Reply:
yes, because default UUID is a random based generator, for PostgreSQL I need UUID v7, it time based, so it can be sorted by b-tree index